assembler/theme.json

```diff
@@ -340,5 +388,19 @@
 			"fontSize": "16px",
 			"lineHeight": "1.65"
 		}
-	}
-}
+	},
+	"templateParts": [
```

Comment:
The template part "comments" is used in single.html, but it's not included in `templateParts`. I remember we had issues when template parts were missing here. 

@alaczek shall we add "comments" here?   

From the [docs](https://developer.wordpress.org/themes/block-themes/templates-and-template-parts/#adding-template-parts-to-theme-json):

> If you would like the template part to be selectable in the block inserter, you also need to include it in the templateParts section of theme.json.

Reply:
I opened a new PR to take care of this.